src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -890,6 +891,7 @@ def acquireTiledArea(streams, stage, area, overlap=0.2, settings_obs=None, log_p
     future.running_subf = model.InstantaneousFuture()
     future._task_lock = threading.Lock()
     # Create a tiled acquisition task
+    # TODO input the focus_range as input argument and do not calculate it in this task
     task = TiledAcquisitionTask(streams, stage, area, overlap, settings_obs, log_path, future=future, zlevels=zlevels,
```

Comment:
add focus range in tiled acquisition and adjust the code withing the tiled acquisition class to use the focus range from the input arguments